notNeededPackages.json

```diff
@@ -3906,6 +3906,12 @@
             "sourceRepoURL": "https://github.com/neutrinojs/webpack-chain",
             "asOfVersion": "5.2.0"
         },
+        {
+            "libraryName": "typescript",
+            "typingsPackageName": "whatwg-streams",
+            "sourceRepoURL": "https://streams.spec.whatwg.org",
+            "asOfVersion": "3.2.1"
```

Comment:
I noticed that in other PRs that removed types for DOM APIs (e.g. #34641), the `asOfVersion` field is always set to `2.0.0`. I'm not sure if there's a special reason for this?

I've put in the TypeScript version in which streams were added to the DOM types. If that's not how this field works, let me know. 🙂